src/librustc/session/config.rs

```diff
@@ -709,6 +709,36 @@ pub fn append_configuration(cfg: &mut ast::CrateConfig,
     }
 }
 
+unsafe fn configure_llvm(sess: &Session) {
```

Comment:
It would be extremely convenient if it was possible to call it from some kind of entry point in `librustc_driver` (instead of `build_configuration`) but I was unable to identify such function.